examples/Remarketing/AddMerchantCenterDynamicRemarketingCampaign.php

```diff
@@ -0,0 +1,454 @@
+        GoogleAdsClient $googleAdsClient,
+        int $customerId,
+        string $adGroupResourceName
+    ) {
+        $marketingImageResourceName = self::uploadAsset(
+            $googleAdsClient,
+            $customerId,
+            'https://goo.gl/3b9Wfh',
+            'Marketing Image'
+        );
+        $logoImageResourceName = self::uploadAsset(
+            $googleAdsClient,
+            $customerId,
+            'https://goo.gl/mtt54n',
+            'Logo Image'
+        );
+
+        // Creates the responsive display ad info object.
+        $responsiveDisplayAdInfo = new ResponsiveDisplayAdInfo([
+            'marketing_images' => [
+                new AdImageAsset([
+                    'asset' => new StringValue(['value' => $marketingImageResourceName])
+                ])
+            ],
+            'square_marketing_images' => [
+                new AdImageAsset([
+                    'asset' => new StringValue(['value' => $logoImageResourceName])
+                ])
+            ],
+            'headlines' => [
+                new AdTextAsset([
+                    'text' => new StringValue(['value' => 'Travel'])
+                ])
+            ],
+            'long_headline' => new AdTextAsset([
+                'text' => new StringValue(['value' => 'Travel the World'])
+            ]),
+            'descriptions' => [
+                new AdTextAsset([
+                    'text' => new StringValue(['value' => 'Take to the air!'])
+                ])
+            ],
+            'business_name' => new StringValue(['value' => 'Interplanetary Cruises']),
+            // Optional: Call to action text.
+            // Valid texts: https://support.google.com/adwords/answer/7005917
+            'call_to_action_text' => new StringValue(['value' => 'Apply Now']),
+            // Optional: Creates a logo image and set it to the ad.
+            'logo_images' => [
+                new AdImageAsset([
+                    'asset' => new StringValue(['value' => $logoImageResourceName])
+                ])
+            ],
+            // Optional: Creates a square logo image and set it to the ad.
+            'square_logo_images' => [
+                new AdImageAsset([
+                    'asset' => new StringValue(['value' => $logoImageResourceName])
+                ])
+            ]
+            // Whitelisted accounts only: Sets color settings using hexadecimal values.
+            // Sets allowFlexibleColor to false if you want your ads to render by always
+            // using your colors strictly.
+            // 'main_color' => new StringValue(['value' => '#0000ff']),
+            // 'accent_color' => new StringValue(['value' => '#ffff00']),
+            // 'allow_flexible_color' => new BoolValue(['value' => false]),
+            // Whitelisted accounts only: Sets the format setting that the ad will be served in.
+            // 'format_setting' => DisplayAdFormatSetting::NON_NATIVE
```

Comment:
There are no mentioning about whitelisting in Simply anymore.
Not sure if this is lifted.